test/integration/http_integration.cc

```diff
@@ -2117,6 +2117,287 @@ void HttpIntegrationTest::testTrailers(uint64_t request_size, uint64_t response_
   }
 }
 
+// Adds metadata inserting filter before metadata consuming filter. Verify metadata are consumed.
+void HttpIntegrationTest::testInsertBeforeConsumeResponseMetadata() {
+  config_helper_.addFilter(R"EOF(
+name: response-metadata-insert-filter
+config: {}
+)EOF");
+
+  config_helper_.addFilter(R"EOF(
+name: response-metadata-consume-filter
+config: {}
+)EOF");
+
+  config_helper_.addConfigModifier(
+      [&](envoy::config::filter::network::http_connection_manager::v2::HttpConnectionManager& hcm)
+          -> void { hcm.set_proxy_100_continue(true); });
```

Comment:
any reason to not just stick this one in initialize?  it won't do harm to have it on for the other tests

Reply:
Fixed by moving to initialize(). Thanks!